deploy/operator/ztp/deploy_sno.sh

```diff
@@ -0,0 +1,89 @@
+export ASSISTED_NAMESPACE="${ASSISTED_NAMESPACE:-assisted-installer}"
```

Comment:
should we rename this file to ``deploy_spoke_cluster.sh``?

Reply:
yes